helm/video/values.yaml

```diff
@@ -173,23 +196,6 @@ services:
       port: 5432
       user: postgres
 
-  - name: "data-broker-producer"
-    type: kafka-producer
-    config:
-      tracing_service_name: data-broker
-      brokers: 
-        - "" # TODO
-      username: "" # TODO
-      password: "" # TODO
-      tls_enable: true
-      sasl_enable: true
-
-  - name: "message-broker-producer"
-    type: rabbitmq-producer
-    config:
-      tracing_service_name: message-broker
-      url: "amqp://guest:guest@message-broker:5672/"
-
```

Comment:
Just a note here on why I removed this:

The `Script: | ...` body of the `background_services:` for "encoder-data-broker-consumer" relies on
```
ctx.get_service("data-broker-producer");
```

so having two separate named definitions for producers (e.g. for Rabbit-MQ vs Kafka) means we would also need to remember to modify that `get_service` call depending on which stack we are using. It is simpler to just modify the configuration details for the particular type of producer we are using under the same service name.

Reply:
The goal of this architecture was to have rabbitmq and kafka for different purposes. Please keep the original architecture.

Reply:
rolled back